build/project-template/internal/nativescript-build.xcconfig

```diff
@@ -6,6 +6,9 @@ OTHER_LDFLAGS[config=Debug] = $(inherited) "$(SRCROOT)/internal/TKLiveSync/TKLiv
 // We need to add CONFIGURATION_BUILD_DIR here so that we can explicitly quote-escape any paths in it, because the implicitly added path by Xcode is not always escaped
 FRAMEWORK_SEARCH_PATHS[sdk=*] = $(inherited) "$(SRCROOT)/internal/" "$(CONFIGURATION_BUILD_DIR)"
 
+// Launch custom linker script in order to generate metadata binary file and link it in the executable
+LD = $SRCROOT/internal/nsld.sh
```

Comment:
Shouldn't be `$(SRCROOT)/internal/nsld.sh` (with `(` and `)` around `SRCROOT`)?

Reply:
Both of them are OK and expand correctly to the Xcode variable.

Reply:
I solved the problem by changing the `LD = $SRCROOT/internal/nsld.sh` line for `NS_LD = $SRCROOT/internal/nsld.sh`